deploy/template.yaml

```diff
@@ -424,6 +493,9 @@ Outputs:
   IPVCriUkPassportInternalAPIGatewayID:
     Description: CRI UK Passport Internal API Gateway ID
     Value: !Ref IPVCriUKPassportInternalAPI
+  IPVCriUkPassportPrivateAPIGatewayID:
```

Comment:
This will need to be exported so that passport-front can `!ImportValue` in the next step, example from core-back here: https://github.com/alphagov/di-ipv-core-back/blob/3ca51761b5f6dc9da7a1b91415cbd6649433b53c/deploy/template.yaml#L931

Reply:
Updated.